custom_components/tuya_local/devices/earu_k6h3aw_thermostat.yaml

```diff
@@ -299,3 +300,16 @@ entities:
           - dps_val: null
             value: false
           - value: true
+  - entity: binary_sensor
```

Comment:
This info is already available from the hvac_action on the climate entity, so this sensor is not needed.